README.md

```diff
@@ -778,6 +779,8 @@ i.e analyzer,alias types are ignored by default
 
 `--ignoreChildError` allows multi-elasticdump to continue if a child throws an error.
 
+`--matchType`  allows multi-elasticdump to fetch index name from elasticsearch alias endpoint of data_streams 
```

Comment:
Can we change to the following?
allows multi-elasticdump to fetch indices from the specified elasticsearch endpoint. Two options are supported - \`alias, datastream\`

Reply:
fixed